src/apps/service/apps/job_listings.clj

```diff
@@ -104,7 +121,9 @@
      :total     (count-jobs user params types analysis-ids)}))
 
 (defn admin-list-jobs-with-external-ids [external-ids]
-  {:analyses (mapv format-admin-job (jp/list-jobs-by-external-id external-ids))})
+  (let [jobs      (jp/list-jobs-by-external-id external-ids)
+        rep-steps (group-by (some-fn :parent_id :job_id) (jp/list-representative-job-steps (mapv :id jobs)))]
```

Comment:
This is another thing that probably won't cause a problem because (IIRC) the DE disallows the use of interactive apps in HT batches. With `rep-steps` grouped by `(some-fn :parent_id :job_id)`, however, the result would be that the parent job in a batch would have the URLs of all of the child jobs associated with it, but none of the child jobs would have any URLs associated with them. In this case, I think you can get away with simply grouping by `:job_id` in the same way that `list-job` does.

Reply:
Ah, another good catch.  I think we did decide we wouldn't allow HT batches, but I want this to be correct in case we do.  I'll update it - thanks!